gitpractice/javascript/task7.js

```diff
@@ -0,0 +1,15 @@
+/*http://www.codewars.com/kata/partition-on*/
+function partitionOn(pred, items) {
+    var evenNumberArray = items.filter(pred);
+    var oddNumberArray = items.filter(item => {
+        if (pred(item) == false)
+            return true;
+        else
+            return false;
+    })
```

Comment:
Here, you don't need to have a if else, since you are using filter method - you just need to specify the condition, which itself will return true or false.

look into array method filter documentation, for more understanding of the method.